azure-pipelines.yml

```diff
@@ -449,12 +449,12 @@ jobs:
       echo "== Testing pip install on Python 3.9 =="
       docker run \
         --rm -v $PYPI_FILES:/mnt/pypi python:3.9 \
-        /bin/bash -c "ls /mnt/pypi && pip install -f /mnt/pypi -q azure-cli==$CLI_VERSION.* && az self-test && az --version && sleep 5"
+        /bin/bash -c "cd /mnt/pypi && ls && pip install -f ./ azure_cli-$CLI_VERSION*whl && az self-test && az --version && sleep 5"
```

Comment:
Why cd into `/mnt/pypi`? Is `-f ./` still necessary now?

Reply:
I don't want to write full path in `-f` and whl file.

Yes, the `azure_cli-$CLI_VERSION*whl` requires `azure-cli-core-XX.post`, we need to add this folder into additional repositories.